src/index.html

```diff
@@ -4,9 +4,311 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Smartphones</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Gadgets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Tablets</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Photo</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card card--location" data-qa="card">
```

Comment:
Yes, if you are consistently using the `--location` modifier with the `card` block, it may be better to incorporate its behavior directly into the main `card` block. This approach can help reduce redundancy and simplify your code by making the `location` style or functionality a default part of the `card` block.

In general, if a modifier is always present, it’s often a sign that it could be merged into the base block to streamline the design and avoid unnecessary complexity.